src/views/LoanDetails/LoanDetails.test.js

```diff
@@ -304,12 +320,12 @@ describe('LoanDetails', () => {
       });
       expect(screen.getAllByText('ui-users.loans.claimReturned')).toBeTruthy();
     });
-    it.skip('When props ID and proxy ID are same with claims', () => {
+    it('When props ID and proxy ID are same with claims', () => {
```

Comment:
This title seems to be a suitable title for a describe block. what is being tested in the scope of this unit test?